napari/_vispy/vendored/volume.py

```diff
@@ -39,8 +39,8 @@
 from vispy.visuals import Visual
```

Comment:
I'm guessing some of the diff in this file is updating the vendored volume to the latest on vispy main, and then some are your changes needed for this PR. Maybe you make a PR just updating the vendored volume that we can merge quickly and then this diff will be easier to review. Also I'm wondering if your thinking about a PR to add this functionality to vispy upstream (apologies if this has already been discussed/ linked too, but if so re-linking again here would be great, thanks!!)

Reply:
There is an [active PR in vispy](https://github.com/vispy/vispy/pull/2074), but I'm not sure if it's up to date with this one.

Reply:
Ok great - as long as we're keeping the API changes in sync, implementation details can diverge as we get the stuff in!!

Reply:
I'll add a separate PR updating us to the current vispy main and hopefully the diff will get cleaner!